app/scripts/modules/core/pipeline/config/stages/travis/travisExecutionDetails.controller.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+
+import {EXECUTION_DETAILS_SECTION_SERVICE} from 'core/delivery/details/executionDetailsSection.service';
+
+let angular = require('angular');
+
+module.exports = angular.module('spinnaker.core.pipeline.stage.travis.executionDetails.controller', [
+  require('angular-ui-router'),
+  EXECUTION_DETAILS_SECTION_SERVICE,
+  require('core/delivery/details/executionDetailsSectionNav.directive.js'),
+])
+  .controller('TravisExecutionDetailsCtrl', function ($scope, $stateParams, executionDetailsSectionService) {
+
+    $scope.configSections = ['travisConfig', 'taskStatus'];
+
+    let initialized = () => {
+      $scope.detailsSection = $stateParams.details;
+      getFailureMessage();
+    };
+
+    function getFailureMessage() {
+      let failureMessage = $scope.stage.failureMessage,
+          context = $scope.stage.context || {},
+          buildInfo = context.buildInfo || {},
+          testResults = buildInfo && buildInfo.testResults && buildInfo.testResults.length ?
+            buildInfo.testResults : [],
+          failingTests = testResults.filter(function(results) { return results.failCount > 0; }),
+          failingTestCount = failingTests.reduce(function(acc, results) {
+            return acc + results.failCount;
+          }, 0);
+
+      if (buildInfo.result === 'FAILURE') {
+        failureMessage = 'Build failed.';
+      }
+
+      if (failingTestCount) {
+        failureMessage = failingTestCount + ' test(s) failed.';
```

Comment:
This is how it is done in the Jenkins implementation. I guess you'll have to visit Travis/Jenkins to get the details.